torch/onnx/utils.py

```diff
@@ -480,13 +493,22 @@ def _model_to_graph(model, args, verbose=False,
                             module=module)
     from torch.onnx.symbolic_helper import _onnx_shape_inference
     if isinstance(model, torch.jit.ScriptModule) or isinstance(model, torch.jit.ScriptFunction):
-        assert example_outputs is not None, "example_outputs must be provided when exporting a ScriptModule or " \
-                                            "ScriptFunction."
-        if isinstance(example_outputs, list):
-            example_outputs = [example_outputs]
+        if example_outputs is None:
+            example_outputs = _get_example_outputs(model, args)
+        else:
+            # example_outpus specified
```

Comment:
@hwangdeyu updated code to replicate previous results.

Reply:
Looks great to me!